### PR TITLE
Updates PR Template

### PR DESCRIPTION
**Pull request by matthewschlue-wf**
Update the PR template to more accurately reflect our requirements for contribution

@Workiva/messaging-pp @Workiva/product2

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on HipChat: InfoSec Forum._

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
-Pull requests should be made against the 'develop' branch, not master.
+### Story:
+TODO
 
-@Workiva/messaging-pp @Workiva/product2
+### Acceptance Criteria:
+- [ ] At least one InfRe Squad 2 member has reviewed and +1'd
+- [ ] Code has been tested and results documented
```
Comment (garrickpeterson-wf) [inline .github/PULL_REQUEST_TEMPLATE.md]: Should probably include unit test updates.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+### Design Notes:
+TODO
+
+### How To Test:
+TODO
+
+### My Test Results:
+TODO
+
+#### Reviewers: @
```
Comment (yuanmwang-wf) [inline .github/PULL_REQUEST_TEMPLATE.md]: Remove extra `@` here?

Comment (matthewschlue-wf) [reply .github/PULL_REQUEST_TEMPLATE.md]: addressed

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (matthewschlue-wf) [reply .github/PULL_REQUEST_TEMPLATE.md]: addressed

**Comment by yuanmwang-wf** [comment]
+1

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,3 +1,22 @@
-Pull requests should be made against the 'develop' branch, not master.
+### Story:
+TODO
 
-@Workiva/messaging-pp @Workiva/product2
+### Acceptance Criteria:
+- [ ] At least one InfRe Squad 2 member has reviewed and +1'd
+- [ ] Code has been tested and results documented
+- [ ] Unit tests have been updated
+- [ ] Updates to documentation and change logs if necessary
```
Comment (brianshannan-wf) [inline .github/PULL_REQUEST_TEMPLATE.md]: IMO remove "and change logs" as we don't have a change log

Comment (matthewschlue-wf) [reply .github/PULL_REQUEST_TEMPLATE.md]: addressed

**Comment by yuanmwang-wf** [comment]
+1

**Comment by brianshannan-wf** [comment]
+1

**Comment by yuanmwang-wf** [comment]
@Workiva/release-management-p

**Comment by garrickpeterson-wf** [comment]
+1

**Pull request merged**